tests/backup/backup_resiliency_test.go

```diff
@@ -1558,3 +1558,200 @@ var _ = Describe("{ScaleDownPxBackupPodWhileBackupAndRestoreIsInProgress}", func
+			log.FailOnError(err, fmt.Sprintf("Fetching [%s] cluster status", destinationClusterName))
+			dash.VerifyFatal(destClusterStatus, api.ClusterInfo_StatusInfo_Online, fmt.Sprintf("Verifying if [%s] cluster is online", destinationClusterName))
+		})
+		Step("Getting the total number of worker nodes in source and destination cluster", func() {
+			log.InfoD("Getting the total number of worker nodes in source and destination cluster")
+			sourceClusterWorkerNodes = node.GetWorkerNodes()
+			log.InfoD("Total number of worker nodes in source cluster are %v", len(sourceClusterWorkerNodes))
+			totalNumberOfWorkerNodes = append(totalNumberOfWorkerNodes, sourceClusterWorkerNodes...)
+			log.InfoD("Switching cluster context to destination cluster")
+			SetDestinationKubeConfig()
+			destinationClusterWorkerNodes = node.GetWorkerNodes()
+			log.InfoD("Total number of worker nodes in destination cluster are %v", len(destinationClusterWorkerNodes))
+			totalNumberOfWorkerNodes = append(totalNumberOfWorkerNodes, destinationClusterWorkerNodes...)
+			log.InfoD("Total number of worker nodes in source and destination cluster are %v", len(totalNumberOfWorkerNodes))
+			log.InfoD("Switching cluster context backup to source cluster")
+			err := SetSourceKubeConfig()
+			log.FailOnError(err, "Switching context to source cluster")
+		})
+		Step("Verifying the license count after adding source and destination clusters", func() {
+			log.InfoD("Verifying the license count after adding source and destination clusters")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			licenseInspectResponse, err = Inst().Backup.InspectLicense(ctx, licenseInspectRequestObject)
+			log.FailOnError(err, "Getting the license for organisation %v", orgID)
+			licenseResponseInfoFeatureInfo = licenseInspectResponse.GetLicenseRespInfo().GetFeatureInfo()
+			dash.VerifyFatal(licenseResponseInfoFeatureInfo[0].Consumed, int64(len(totalNumberOfWorkerNodes)), fmt.Sprintf("Verifying the license count when source cluster with %d worker nodes and destination cluster with %d worker nodes are added to backup", len(sourceClusterWorkerNodes), len(destinationClusterWorkerNodes)))
+		})
+		Step("Applying label portworx.io/nobackup=true to worker nodes and verifying the license count", func() {
+			log.InfoD("Applying label portworx.io/nobackup=true to one of the worker node on source cluster and verifying the license count")
+			err := Inst().S.AddLabelOnNode(sourceClusterWorkerNodes[0], "portworx.io/nobackup", "true")
+			if err == nil {
+				sourceClusterWorkerNodeLabelApplied = true
+				sourceClusterWorkerNodeLabelRemoved = false
+			}
+			log.FailOnError(err, fmt.Sprintf("Failed to apply label portworx.io/nobackup=true to worker node %v", sourceClusterWorkerNodes[0].Name))
+			log.Infof("Sleeping for 10 minutes for license count to get updated")
+			time.Sleep(10 * time.Minute)
```

Comment:
Why do we need sleep and timeout both?

Reply:
10 minutes in general it takes to get updated. So did not want to keep retrying again and again
Added retry so that in case of it takes more than 10 minutes, test case should not fail because of timing issue.
Is that ok or should I remove sleep and just add timeout?

Reply:
If you are pretty sure that it will take 10 mins it's okay to do so.

Reply:
Will once confirm with dev about the time taken to update the license.
Will add sleep and timeout accordingly

Reply:
Changed it:
1)removed 10 minutes sleep
2)Added timeout of 15 minuted with retry after every 1 min